### ISSUE TITLE
Support for new official application, 'Smart eLife'

### ISSUE THREAD

**Issue by OrigamiDream** [OrigamiDream/homebridge-daelim-smarthome#6]
DL E&C Co.,Ltd. 에서 [Smart eLife](https://apps.apple.com/kr/app/%EC%8A%A4%EB%A7%88%ED%8A%B8-elife/id1551248421)라는 이름의 새 앱을 배포했습니다.

해당 앱은 다음의 새로 지어진 아파트만을 지원합니다:
- e편한세상 운정어반프라임
- e편한세상 김포로얄 하임
- e편한세상 감일에코앤 
- e편한세상 백련산
- e편한세상 여주

이 앱은 현재 [스마트홈 2.0](https://apps.apple.com/kr/app/e편한세상-스마트홈-2-0/id1109202165)과 동일한 프로토콜을 공유하고 있는지 여부를 알 수 없는 상태입니다.
그러므로, 관련된 내용이 README에 언급되어야 합니다.

**Comment by OrigamiDream** [comment]
@dongs0104  `스마트 eLife` 앱을 사용하는 아파트는 현재 서버 IP 주소를 확인할 수 있는 방법이 없습니다.

**Comment by elvenwhite** [comment]
`스마트 eLife` 앱에 로그인 후 패킷 스니핑으로 서버 IP를 확인하면 될까요?

**Comment by OrigamiDream** [comment]
@elvenwhite  로그인이 가능한 경우면 서버 IP 확인 가능합니다

**Comment by OrigamiDream** [comment]
만약 모든 요청이 HTTP 위에서 동작한다면 기존 스마트홈 2.0과 다른 프로토콜을 사용하고 있기 때문에 아예 따로 추가적인 분석이 필요해집니다.
제가 그 아파트로 이사를 가거나, 다른 분이 계정을 빌려주시거나 하는게 아니면 작업이 힘들겠네요.

**Comment by elvenwhite** [comment]
네 https 파싱해보니 말씀하신대로 모든 요청이 http 위에서 동작하는 것으로 확인됩니다. `README`에 등록하고 별도의 플러그인으로 개발해야 하겠네요. 감사합니다.

**Comment by OrigamiDream** [comment]
`homebridge-daelim-smarthome v1.5.0`이 정식 배포되었습니다.
